root/static/styles/favicons.less

```diff
@@ -34,6 +34,7 @@
 .favicon("genius");
 .favicon("googleplus", 32);
 .favicon("ibdb");
+.favicon("IMDb");
```

Comment:
@Freso , what about "iTunes" and "SMDB"? I'm confused on the capitalizations here.